lang/cs/Org.Apache.REEF.Driver/Org.Apache.REEF.Driver.csproj

```diff
@@ -174,6 +174,13 @@ under the License.
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
+  <Import Project="..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets" Condition="Exists('..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets')" />
```

Comment:
I am going to go with what @jwang98052  has suggested. 
But I would like to point out my philosophical disagreement here on treating csproj as a text file which has to be maintained by hand. We should treat them as auto generated files from a tool. That said, I understand the practicality of the suggestion.